mangaCode/manga.py

```diff
@@ -0,0 +1,32 @@
+import array
+
+class Manga:
+	def __init__(self, name, description, genre, num_chapters, mangaka):
+		self._name = name
+		self._description = description
+		self._genre = genre
+		self._num_chapters = num_chapters
+		self._mangaka = mangaka
+		self._reviews = []
```

Comment:
¿Las reviews tienen que estar en el mismo objeto?

Reply:
Tras comentarlo por Telegram, hemos decidido que sí, sería lo más sencillo para crear el producto mínimamente viable. Las valoraciones serán solo numéricas (como las clásicas estrellas del 1 al 5 en la mayoría de aplicaciones).

Una vez que se tiene el vector relleno, se pueden definir métodos simples como calcular la media de esas reviews.

Reply:
No, no es lo más sencillo y ni siquiera es una buena práctica. Revisad el diseño dirigido por dominio. Y no hagáis las revisiones por Telegram, sino que justificarlo en base a las HUs e issues correspondientes.